src/APL/saleor-cloud/saleor-cloud-apl.ts

```diff
@@ -95,79 +103,172 @@ export class SaleorCloudAPL implements APL {
   async get(saleorApiUrl: string): Promise<AuthData | undefined> {
     debug("Will fetch data from SaleorCloudAPL for saleorApiUrl %s", saleorApiUrl);
 
-    const response = await fetch(this.getUrlForDomain(saleorApiUrl), {
-      method: "GET",
-      headers: { "Content-Type": "application/json", ...this.headers },
-    }).catch((error) => {
-      debug("Failed to reach API call:  %s", extractErrorMessage(error));
-      debug("%O", error);
+    return this.tracer.startActiveSpan(
+      "Call SaleorCloudAPL GET",
+      {
+        attributes: {
+          saleorApiUrl,
+          [SemanticAttributes.PEER_SERVICE]: OTEL_APL_SERVICE_NAME,
+        },
+        kind: SpanKind.CLIENT,
+      },
+      async (span) => {
+        const response = await fetch(this.getUrlForDomain(saleorApiUrl), {
+          method: "GET",
+          headers: { "Content-Type": "application/json", ...this.headers },
+        }).catch((error) => {
+          debug("Failed to reach API call:  %s", extractErrorMessage(error));
```

Comment:
Why not use

```js
try {
  await fetch()
} catch (error) {
  // ...
}
```

Reply:
1. Its the same
2. It was written like this before, in this PR I only wrapped entire function with `startActiveSpan`, so I don't see value of refactoring at the same time (and I also don't see problem with this syntax)

If we agree that try/catch adds some value, we should probably set this on ESLint level as a rule